app/filter.py

```diff
@@ -277,6 +288,28 @@ def update_styling(self, soup) -> None:
         except AttributeError:
             pass
 
+        # Fix body max with on images tab
```

Comment:
Can you elaborate what's going on here? I'm not sure I follow.

Reply:
The max-width returned to the images tab is 700 (for body tag) which is different on all other tabs (736) so we need to replace it. The html styling is also missing and is needed to have the exact same formatting.